backend/src/apiserver/main.go

```diff
@@ -106,6 +106,13 @@ func main() {
 	}
 	log.SetLevel(level)
 
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
+	defer cancel()
+	err = resourceManager.SyncSwfCrs(ctx)
```

Comment:
Does this need to be run synchronously? Could it be run in a separate goroutine instead?